presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -179,6 +180,15 @@ public SystemSessionProperties(
                         value -> Duration.valueOf((String) value),
                         Duration::toString),
                 new PropertyMetadata<>(
+                        QUERY_MAX_EXECUTION_TIME,
+                        "Maximum execution time of a query",
```

Comment:
Do we want to also change the description of the run time to include queueing time explicitly?